packages/moonstone/Input/Input.js

```diff
@@ -210,7 +210,15 @@ const InputBase = kind({
 		 * @type {String|Number}
 		 * @public
 		 */
-		value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+		value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+		/**
+		 * Add voice control property to input tag
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		voiceProps:  PropTypes.object
```

Comment:
Similar question to `ExpandableItem` on the contents of `voiceProps`.

Reply:
Same to ExpandableItem. I will make seperate ticket.